docs/docsite/rst/user_guide/windows_winrm.rst

```diff
@@ -303,7 +303,7 @@ Some system dependencies that must be installed prior to using Kerberos. The scr
 .. code-block:: shell
 
     # Via Yum (RHEL/Centos/Fedora)
-    yum -y install python-devel krb5-devel krb5-libs krb5-workstation
+    yum -y install gcc python-devel krb5-devel krb5-libs krb5-workstation
 
     # Via Apt (Ubuntu)
     sudo apt-get install python-dev libkrb5-dev krb5-user
```

Comment:
Do you think Ubuntu and derivatives also require `gcc` for `python-devel`? if yes, can you edit this line as well.

Reply:
I wanted to add those because I assumed that it would be needed for all of them but I haven't tried to install kerberos/ansible on those machines so I wasn't sure.

Reply:
This link might help us with that question: https://github.com/requests/requests-kerberos/issues/109